spec/jobs/irs_attempts_events_batch_job_spec.rb

```diff
@@ -41,23 +59,50 @@
       end
 
       it 'batches and writes attempt events to an encrypted file' do
-        result = IrsAttemptsEventsBatchJob.perform_now(start_time, dir_path: @dir_path)
-        expect(result[:file_path]).not_to be_nil
+        # puts "PUBLIC KEY = "
+        # puts Base64.strict_encode64(public_key.to_der)
 
-        file_data = File.open(result[:file_path], 'rb') do |file|
-          file.read
-        end
+        # expect(IrsAttemptsApi::EnvelopeEncryptor).to receive(:encrypt).with(
+        #  data: events.collect{|e| e[:jwe]}.join("\r\n"),
```

Comment:
@Rwolfe-Nava:

Since we were talking about this&mdash;my editor just corrected `events.collect{ |e| e[:jwe }` to `events.pluck(:jwe)` which reads much better. TIL.